net/xtables-addons/patches/500-change-strlcpy-to-strscpy.patch

```diff
@@ -0,0 +1,13 @@
+--- a/extensions/rtsp/nf_conntrack_rtsp.c
++++ b/extensions/rtsp/nf_conntrack_rtsp.c
```

Comment:
However, you described why in the commit description, but this patch lacks those details like who is the author, why it is necessary to have it, etc.  

While thinking about it, why have you decided to separate patch instead of including it in the patch, which adds rtsp extensions? This can not be done as it was in the past like in commit https://github.com/openwrt/packages/commit/4ea6c0065c7474118f9fafe14e5b0518fa4cdd49 ?

Reply:
The rtsp extension patch is not mine.

Reply:
I checked the history for net/xtables-addons what we have here and everyone included it in the first patch as it should be. Having additional patch is really not necessary.